pkg/Apps/Graph/Library/NodeGraphApp.js

```diff
@@ -5,7 +5,7 @@
  * license that can be found in the LICENSE file.
 */
```

Comment:
GraphMaker is the beginning of converting Graph! to CommonApp+Graph format. So, the old one is deprecated and the new one isn't ready yet. :)

Reply:
I'll start on this tomorrow, now that i have a better grasp of the CommonApp+Graph combo!